### PR TITLE
[Icons]: Update icons from Figma

### PR DESCRIPTION
**Pull request by brave-support-admin**
> @brave/leo@0.0.1 update-icons
> node ./src/scripts/update-icons.js

Ignoring icon rainbow-color.svg because:
- has a raster image element
Ignoring icon bitcoin.com-color.svg because:
- has a raster image element
Ignoring icon daum-color.svg because:
- has a raster image element
Ignoring icon openllama-color.svg because:
- has a raster image element

### CODE REVIEW

File: icons/product-leo-size-36px.svg
Comment (fallaciousreasoning) [inline icons/product-leo-size-36px.svg:1]: @aguscruiz these should probably be marked as color icons if we want to maintain the background color

File: icons/product-leo-size-72px.svg
Comment (fallaciousreasoning) [inline icons/product-leo-size-72px.svg:1]: @aguscruiz, these different sized icons all look very similar (I guess the glow is slightly different)

File: icons/product-search-size-36px.svg
Comment (fallaciousreasoning) [inline icons/product-search-size-36px.svg:1]: @aguscruiz same here - these should have a `-color` suffix

File: icons/type-browser.svg
Comment (fallaciousreasoning) [inline icons/type-browser.svg:1]: @aguscruiz `-color`

Comment (fallaciousreasoning) [reply icons/type-browser.svg:1]: Also, this seems like a bit of an odd icon :laughing:

File: icons/type-initials.svg
Comment (fallaciousreasoning) [inline icons/type-initials.svg:1]: I dunno if this needs to be part of Nala :laughing:

Comment (AlanBreck) [reply icons/type-initials.svg:1]: cc @aguscruiz

File: icons/product-leo-size-36px.svg
Comment (aguscruiz) [reply icons/product-leo-size-36px.svg:1]: Will do

File: icons/type-initials.svg
Comment (aguscruiz) [reply icons/type-initials.svg:1]: Nah, probably not. We can ignore that. Should I remove them from this page?

File: icons/type-browser.svg
Comment (aguscruiz) [reply icons/type-browser.svg:1]: Yeah it shouldn't be here

File: icons/type-initials.svg
Comment (AlanBreck) [reply icons/type-initials.svg:1]: Yeah, that way we don't get it automatically included.

File: icons/product-leo-size-72px.svg
Comment (AlanBreck) [reply icons/product-leo-size-72px.svg:1]: @aguscruiz, nudging on ☝️

**Comment by aguscruiz** [comment]
> LGTM!
> 
> Noting that there are multiple removals. Not using any in my projects, but calling out in case their being used any where in the browser. cc @fallaciousreasoning @petemill

I think the ones I removed aren't really used anywhere. Let me know if you found any instance, but I'm pretty sure they're unused

**Review by fallaciousreasoning — APPROVED** [review]
LGTM - looks like brave-core has a dependency on `sync` but I don't think it's actually using it :smile:

**Pull request merged**